test/runtests.jl

```diff
@@ -127,4 +127,18 @@ l = @__LINE__
     def, line = definition(String, @which d_34(1, 2))
     @test line == l - 3
     @test def == "(d_34)(x::T, y::T) where {T<:Number} = x === y"
+end
+
+function g()
+    Base.@_inline_meta
+    print("hello")
+end
+@testset "inline macros" begin
+    def, line = CodeTracking.definition(String, @which g())
+    @test def == """
+    function g()
+        Base.@_inline_meta
+        print("hello")
+    end
+    """
```

Comment:
Should there really be a trailing newline here?

Reply:
I would generalize the test so that we don't really care (`chomp` both the LHS and RHS).

Reply:
Actually, maybe that's a bad idea. Let's strip final line endings as a matter of policy. PR coming.